<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawGridSet</class>
 <widget class="QDialog" name="DrawGridSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>15</y>
     <width>741</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">background-color: green;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="Lb_Para1">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>未知：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="CbShow1">
      <property name="text">
       <string>显示曲线</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="Lb_BT201_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>缩放系数</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinSF1">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="Lb_BT201_3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>纵向偏移</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QDoubleSpinBox" name="doubleSpinPY1">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="btnDel1_1">
      <property name="text">
       <string>删除曲线</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_2">
      <property name="styleSheet">
       <string notr="true">background-color: red;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="Lb_Para2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>未知：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="CbShow2">
      <property name="text">
       <string>显示曲线</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="Lb_BT201_5">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>缩放系数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinSF2">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="Lb_BT201_4">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>纵向偏移</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QDoubleSpinBox" name="doubleSpinPY2">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="btnDel1_2">
      <property name="text">
       <string>删除曲线</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line_3">
      <property name="styleSheet">
       <string notr="true">background-color: blue;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="Lb_Para3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>未知：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="CbShow3">
      <property name="text">
       <string>显示曲线</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="Lb_BT201_7">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>缩放系数</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinSF3">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="Lb_BT201_6">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>纵向偏移</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QDoubleSpinBox" name="doubleSpinPY3">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="btnDel1_3">
      <property name="text">
       <string>删除曲线</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="Line" name="line_4">
      <property name="styleSheet">
       <string notr="true">background-color: yellow;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="Lb_Para4">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>未知：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="CbShow4">
      <property name="text">
       <string>显示曲线</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="Lb_BT201_9">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>缩放系数</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinSF4">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="Lb_BT201_8">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>纵向偏移</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QDoubleSpinBox" name="doubleSpinPY4">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="btnDel1_4">
      <property name="text">
       <string>删除曲线</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="Line" name="line_5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 255);</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="Lb_Para5">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>未知：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="CbShow5">
      <property name="text">
       <string>显示曲线</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="Lb_BT201_11">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>缩放系数</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinSF5">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="Lb_BT201_10">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>纵向偏移</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QDoubleSpinBox" name="doubleSpinPY5">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QPushButton" name="btnDel1_5">
      <property name="text">
       <string>删除曲线</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="Line" name="line_6">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 255);</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="Lb_Para6">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>未知：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QCheckBox" name="CbShow6">
      <property name="text">
       <string>显示曲线</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="Lb_BT201_13">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>缩放系数</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinSF6">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="Lb_BT201_12">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>纵向偏移</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QDoubleSpinBox" name="doubleSpinPY6">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QPushButton" name="btnDel1_6">
      <property name="text">
       <string>删除曲线</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="Line" name="line_7">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="Lb_Para7">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>未知：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QCheckBox" name="CbShow7">
      <property name="text">
       <string>显示曲线</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="Lb_BT201_15">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>缩放系数</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinSF7">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLabel" name="Lb_BT201_14">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>纵向偏移</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QDoubleSpinBox" name="doubleSpinPY7">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QPushButton" name="btnDel1_7">
      <property name="text">
       <string>删除曲线</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="Line" name="line_8">
      <property name="styleSheet">
       <string notr="true">background-color:black;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="Lb_Para8">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>未知：</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QCheckBox" name="CbShow8">
      <property name="text">
       <string>显示曲线</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="Lb_BT201_17">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>缩放系数</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinSF8">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="Lb_BT201_16">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>纵向偏移</string>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QDoubleSpinBox" name="doubleSpinPY8">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QPushButton" name="btnDel1_8">
      <property name="text">
       <string>删除曲线</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DrawGridSet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DrawGridSet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
